tests/test_asset_connect.py

```diff
@@ -29,7 +29,7 @@ def test_storage_connection(self):
 
         conn = S3Connection()
         try:
-            bucket = conn.get_bucket(self.settings['aws_storage_bucket'])
+            bucket = conn.get_bucket(self.settings['veda_s3_hotstore_bucket'])
```

Comment:
@yro can we have settings at different level, like separate settings for development, sandbox and production environment. This way we'll be using the particular setting without worrying about the bucket names.